wgpu/src/api/surface_texture.rs

```diff
@@ -67,6 +67,8 @@ pub enum SurfaceError {
     Lost,
     /// There is no more memory left to allocate a new frame.
     OutOfMemory,
+    /// Acquiring a texture failed for an unknown reason
+    Other,
```

Comment:
Similar'ish suggestion `SurfaceStatus::Unknown`: It's not really an unknown reason, we just haven't passed it through (yet?) 🤔 

```suggestion
    /// Acquiring a texture failed with a generic error. Check error callbacks for more information.
    Other,
```

string below needs adjusting as well.